runelite-client/src/main/java/net/runelite/client/plugins/runecraft/RunecraftPlugin.java

```diff
@@ -103,121 +97,118 @@
 	@Inject
 	private AbyssOverlay abyssOverlay;
 
+	@Inject
+	private RunecraftOverlay runecraftOverlay;
+
 	@Inject
 	private RunecraftConfig config;
 
 	@Inject
 	private Notifier notifier;
 
 	@Provides
-	RunecraftConfig getConfig(ConfigManager configManager)
-	{
+	RunecraftConfig getConfig(ConfigManager configManager) {
 		return configManager.getConfig(RunecraftConfig.class);
 	}
 
 	@Override
-	protected void startUp() throws Exception
-	{
+	protected void startUp() throws Exception {
 		overlayManager.add(abyssOverlay);
 		abyssOverlay.updateConfig();
+		overlayManager.add(runecraftOverlay);
 	}
 
 	@Override
-	protected void shutDown() throws Exception
-	{
+	protected void shutDown() throws Exception {
 		overlayManager.remove(abyssOverlay);
 		abyssObjects.clear();
 		darkMage = null;
 		degradedPouchInInventory = false;
+		overlayManager.remove(runecraftOverlay);
 	}
 
 	@Subscribe
-	public void onConfigChanged(ConfigChanged event)
-	{
+	public void onConfigChanged(ConfigChanged event) {
 		abyssOverlay.updateConfig();
 	}
 
 	@Subscribe
-	public void onChatMessage(ChatMessage event)
-	{
-		if (event.getType() != ChatMessageType.GAMEMESSAGE)
-		{
+	public void onChatMessage(ChatMessage event) {
+		if (event.getType() != ChatMessageType.GAMEMESSAGE) {
 			return;
 		}
 
-		if (config.degradingNotification())
-		{
-			if (event.getMessage().contains(POUCH_DECAYED_MESSAGE))
-			{
+		if (config.degradingNotification()) {
+			if (event.getMessage().contains(POUCH_DECAYED_MESSAGE)) {
 				notifier.notify(POUCH_DECAYED_NOTIFICATION_MESSAGE);
 			}
 		}
 	}
 
 	@Subscribe
-	public void onMenuEntryAdded(MenuEntryAdded entry)
-	{
-		if (!wearingTiara)
-		{
+	public void onMenuEntryAdded(MenuEntryAdded entry) {
+		if (client.getGameState() != GameState.LOGGED_IN) {
+			return;
+		}
+
+		Widget loginScreenOne = client.getWidget(WidgetInfo.LOGIN_CLICK_TO_PLAY_SCREEN);
+		Widget loginScreenTwo = client.getWidget(WidgetInfo.LOGIN_CLICK_TO_PLAY_SCREEN_MESSAGE_OF_THE_DAY);
+
+		if (loginScreenOne != null || loginScreenTwo != null) {
```

Comment:
everything  above thsi isnt really needed, also why remove the wearing tiara switch? now it's always on and not checking for config or anything

Reply:
I figured if the plugin config option was switched on then it would be ok? I can see why the tiara/cape thing is a better idea.